mitiq/cdr/test_clifford_training_data.py

```diff
@@ -0,0 +1,245 @@
+import cirq
```

Comment:
Did you mean to delete `cdr_tests.py`? There is overlapping code between this file and that one. This one looks much better in terms of unit tests.

Reply:
Yeah, basically cdr_tests.py is more code i have been using to look at the outputs of circuits ect. More involved testing. I will remove it.